tests/fixtures/misc/checker/expm1.py

```diff
@@ -0,0 +1,9 @@
+import torch
+a = torch.randn(5)
+b = torch.exp(a) - 1
+c = torch.exp(a) - 1.0
```

Comment:
Technically people could write `- 1 + torch.exp(a)`, although very rare. We can add it, or keep it as false negative.

Reply:
I'll add false negative test.
Handling `- 1 + torch.exp(a)` will complicate implementation, and it should be very rare.